lib/delivery-locations-resolver.js

```diff
@@ -175,6 +175,15 @@ class DeliveryLocationsResolver {
     }
   }
 
+  static sortPosition (location) {
+    if (location.deliveryLocationTypes.includes('Scholar')) return 0
+    if (location.code === 'mal') return 1
+    if (location.code.startsWith('ma')) return 2
+    if (location.code.startsWith('pa') || location.code.startsWith('my') || location.code.startsWith('lp')) return 3
```

Comment:
i think lp is only used for circulating material holding locations at lpa. probably not necessary here. 
also, can we put schomburg above lpa? at office hours a couple months ago a librarian mentioned that it feels like schomburg is always at the bottom of search results. might be nice to not have it at the bottom here. not sure if there was an official order given with the acceptance criteria.

Reply:
When I tested this there was a location with code `lp` so I think we should include it. I can try to find the item if you like.
The ticket specifies LPA then Schomburg so I think we should go with that but we can raise the issue

Reply:
oh ok! just follow the spec. i remember it sounding like there wasn't a fixed preference.

i'd be interested in seeing what is the location or customer code that is returning that delivery location. I don't see any `lp*` location codes in the `deliverableTo` columns in nypl core csvs. NP appears to be the only lpa code in the recap customer code csv, but that corresponds to sierra location `par`.